cli/config/configuration-acrn.toml

```diff
@@ -0,0 +1,368 @@
+# Copyright (c) 2017-2019 Intel Corporation
```

Comment:
this file is auto-generated, can you remove it?

Reply:
> @vijaydhanraj power is failing in travis, could you add the equivalent or dummy (if not supported or to be added in the future ) implementation like you did in `acrn_amd64.go`

Instead of adding dummy functions/ interfaces for each arch, should I move the file to vendor/govmm/intel/acrn. Please suggest.

Reply:
Hi @vijaydhanraj - I think you are right - the acrn support should be added to https://github.com/intel/govmm and then the runtime can leverage acrn support via the vendored in version of govmm. But let's see what @markdryan thinks.

Reply:
Hi @markdryan any thoughts on adding virtcontainers/acrn_amd64.go file to vendor/govmm/intel/acrn to avoid adding dummy functions to for unsupported architecture?

Reply:
That doesn't seem correct to me.

Reply:
Just spoke with @amshinde. Will remove acrn_amd64.go as there is nothing specific to amd64. Will add the code to acrn_arch_base.go.

Reply:
Are you still interested in adding acrn support to govmm?  If so, I need a bit more context and an explanation of what exactly would be added and how.  For example, does acrn use qmp or does it have a totally different protocol?

Reply:
Hi @markdryan, I don't think the support is needed. But I do use "acrnctl" (this another binary) for interacting with the ACRN-DM. It works using sockets underneath but this is abstracted from the user (application). The user(or app) just uses the command it exposes. For example to hot-plug container rootfs, the command will be acrnctl blkrescan vm_name slot,newpath. (similar to Firecracker use case)